README.md

```diff
@@ -0,0 +1,194 @@
+<a name="readme-top"></a>
+
+<img src="./src/images/myLogo.png"></img>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [FAQ]
+- [📝 License](#license)
+
+
+
+# 📖 [COVID 19 MAP METRICS] <a name="about-project"></a>
+
+> 
+**COVID 19 MAP METRICS** is a project i worked on in respect of all lives lost during the COVID 19 pandemic. This application helps in checking of total global statistics of infected people and lost lives per country and region/state.
+
+## 🛠 Built With <a name="built-with">
+> Visual Studio Code and Github
+### Tech Stack <a name="tech-stack">
+
+> The Tech stack used in this project is the frontend technology 'REACT.js'
+<details>
+  <summary>Cascading Style Sheet</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>REACT</summary>
+  <ul>
+    <li><a href="https://react.dev/">REACT</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Fresh quality content]**
+- **[Speed and responsive]**
+- **[Easy to use]**
+- **[Smooth transitions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live-Demo]("https://mapmetrics.onrender.com/")
+> [Loom Presentation]("https://www.loom.com/share/1dbc6515012342daab42af84bf1180d1")
```

Comment:
- [ ] Great work including the Live demo link and video presentation, kindly make sure that the links are accessible from the README file, as for now I can't access them from the README